app/src/main/java/bg/zahov/app/ui/workout/WorkoutScreen.kt

```diff
@@ -68,9 +68,76 @@ import androidx.lifecycle.compose.collectAsStateWithLifecycle
 import androidx.lifecycle.viewmodel.compose.viewModel
 import bg.zahov.app.data.model.SetType
 import bg.zahov.app.ui.theme.FitnessTheme
+import bg.zahov.app.ui.workout.add.AddTemplateWorkoutViewModel
 import bg.zahov.fitness.app.R
 import kotlinx.coroutines.delay
 
+
+@Composable
+fun AddTemplateWorkoutScreen(
+    addTemplateViewModel: AddTemplateWorkoutViewModel = viewModel(),
+    onAddExercise: () -> Unit,
+    onReplaceExercise: () -> Unit,
+    onBackPressed: () -> Unit,
+    onCancel: () -> Unit,
+) {
+    /**
+     * In order to center the items of 2 independent rows we need to have
+     * pre-defined weight values to ensure consistency
+     */
+    val weightValues = rememberSaveable { arrayOf(1f, 1f, 2f, 2f) }
```

Comment:
Doesn't need to be in a `rememberSaveable`

Reply:
I still see a `remember` being used. Why is that?

Reply:
I messed up on the commits and it seems to have split up the changes because I commited too early and didn't do a second push